app/assets/stylesheets/shared_accesses.scss

```diff
@@ -0,0 +1,8 @@
+#shared-access-empty {
+  .user-circle {
+    width: 100px;
+    height: 100px;
+    background-color: $primary-light;
+    border-radius: 50%;
```

Comment:
We could replace the background-color and border-radius with a Bootstrap class.
But we would need to add style to the div or add a custom class for the width and height.

Reply:
Yeah ideally move as much as possible to classes using Bootstrap. CSS should be the last resort

Reply:
After review, this can't be done strictly with Bootstrap classes.
Could be done with style={}, but React docs recommend using a CSS class.
https://reactjs.org/docs/dom-elements.html#style